Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1381,6 +1381,17 @@ theorem lt_ord_succ_card (o : Ordinal) : o < (succ o.card).ord :=
   lt_ord.2 <| lt_succ _
 #align cardinal.lt_ord_succ_card Cardinal.lt_ord_succ_card
 
+/--
+A variation on `Cardinal.lt_ord` using `≤`: If `i` is no greater than the
+initial ordinal of cardinality `κ`, then the cardinal of any of its realizations
+is no greater than `κ`.
+
+The converse, however, is false (for instance, `i = ω+1` and `κ = ℵ₀`).
+-/
+lemma card_le_of_le_ord {κ : Cardinal} {i : Ordinal} (hi : i ≤ κ.ord) :
+  i.card ≤ κ := by
```

Comment:
Should I add that `_iff`?

Reply:
You're welcome to! It may be better to derive this from that stronger result.

Reply:
> You're welcome to! You just need to combine `Order.lt_succ_iff` and `Cardinal.lt_ord`, I think. It may be better to derive `card_le_of_le_ord` from that stronger result.

I added this `card_le_iff`, but I'm not sure how to derive the older result from this (the present proof is rather short; and the statement, simple enough).